packages/components/src/lib/controllers/app-controller.js

```diff
@@ -968,6 +956,15 @@ if you want to create a different configuration of an app controller.
     // Inject Alpheios CSS rules
     this.activateOnPage()
 
+    this.isActivated = true
+    this.isDeactivated = false
+
+    this.activateModules()
+    // this.state.activate()
```

Comment:
why is this commented out?

Reply:
I just forgot to remove it. It has been moved to the UI controller and is no longer needed: https://github.com/alpheios-project/alpheios-core/blob/dev-ui-controller-refactoring-3/packages/components/src/lib/controllers/ui-controller.js#L131. Will clean it up